shared/src/main/scala/Elements.scala

```diff
@@ -259,20 +261,46 @@ object Elements:
       case (a: String, b: String) => a == b
     }
 
-    val exec = addElem(
-      Monad,
+    val exec = addDirect(
       "Ė",
       "Execute lambda | Evaluate as Vyxal | Power with base 10",
-      List("execute-lambda", "evaluate-as-vyxal", "power-base-10"),
+      List("execute-lambda", "evaluate-as-vyxal", "power-base-10", "call", "@"),
+      Some(1),
       "a: fun -> Execute a",
       "a: str -> Evaluate a as Vyxal",
       "a: num -> 10 ** n"
-    ) {
-      case fn: VFun => Interpreter.executeFn(fn)
-      case code: String =>
-        Interpreter.execute(code)
-        summon[Context].pop()
-      case n: VNum => 10 ** n
+    ) { ctx ?=>
+      ctx.pop() match
+        case fn: VFun =>
+          ctx.push(Interpreter.executeFn(fn))
+          if fn.arity == -1 then
+            ctx.pop() // Handle the extra value pushed by lambdas that operate on the stack
+        case code: String => Interpreter.execute(code)
+        case x            => ctx.push(execHelper(x))
+    }
+
+    def execHelper(value: VAny)(using ctx: Context): VAny =
+      value match
+        case code: String =>
+          Interpreter.execute(code)
+          ctx.pop()
+        case n: VNum     => 10 ** n
+        case list: VList => list.vmap(execHelper)
+        case _ => throw new Exception("Can't exec on functions in lists")
```

Comment:
If we can exec code in lists, why not functions too?

Reply:
Because it was 12am when I made the `execHelper` function and I was way too tired to handle the function case properly.